grails-app/i18n/messages.properties

```diff
@@ -516,6 +540,10 @@ default.success.message=Success
 default.summary.label=Summary
 default.support.label=Help
 default.systemError.label=System error
+default.dataTable.noData.label=No data available in table
+default.dataTable.showingEntries.label=Showing _START_ to _END_ of _TOTAL_ entries
```

Comment:
I neglected to comment on this before. Why are we using this syntax instead of args i.e. {0} {1}? And where / when do these parameters get replaced?

Reply:
These are not the same args as `{0}`, `{1}`, these are specific `datatable` params that need to be provided like this and datatable "decodes" those params and substitute proper numbers into this.
Source:
https://datatables.net/examples/i18n/options.html